linera-views/src/context.rs

```diff
@@ -276,19 +276,16 @@ pub type MemoryContext<E> = ViewContext<E, MemoryStore>;
 /// Provides a `MemoryContext<()>` that can be used for tests.
 #[cfg(with_testing)]
 pub fn create_test_memory_context() -> MemoryContext<()> {
```

Comment:
Can we remove this function then?

Reply:
Yes, we could.
But the function occurs 393 times in the code, so at that size, maybe be prudent.
The replacement is 6 more characters.
If ok, I could replace it in another PR.

Reply:
Next PR sounds good